<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAffichageUtilisateurs</class>
 <widget class="QDialog" name="DlgAffichageUtilisateurs">
  <property name="windowTitle"><string>Affichage des utilisateurs</string></property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="filterLayout">
     <item><widget class="QComboBox" name="comboFiltre">
       <item><property name="text"><string>Tous</string></property></item>
       <item><property name="text"><string>Actifs</string></property></item>
       <item><property name="text"><string>Inactifs</string></property></item>
     </widget></item>
     <item><widget class="QPushButton" name="btnActualiser"><property name="text"><string>Actualiser</string></property></widget></item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="table">
     <property name="columnCount"><number>4</number></property>
     <column><property name="text"><string>Nom</string></property></column>
     <column><property name="text"><string>Email</string></property></column>
     <column><property name="text"><string>Mot de passe</string></property></column>
     <column><property name="text"><string>Actif</string></property></column>
    </widget>
   </item>
   <item><widget class="QPushButton" name="btnFermer"><property name="text"><string>Fermer</string></property></widget></item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
